collector/consts.go

```diff
@@ -19,12 +19,8 @@ const (
 )
 
 var (
-	// blxDefaultURL options - via https://docs.bloxroute.com/introduction/cloud-api-ips
-	// wss://virginia.eth.blxrbdn.com/ws
-	// wss://uk.eth.blxrbdn.com/ws
-	// wss://singapore.eth.blxrbdn.com/ws
-	// wss://germany.eth.blxrbdn.com/ws
-	blxDefaultURL = common.GetEnv("BLX_URI", "wss://virginia.eth.blxrbdn.com/ws")
+	// Bloxroute URL - should point to local Gateway GRPC port
+	blxDefaultURL = common.GetEnv("BLX_URI", "127.0.0.1:1001")
```

Comment:
i'm unsure about the best way for this, whether to keep the WebSocket default URL because  it works without any additional services and is clearly explained in the bloxroute docs 🤔